source/library/Witch/Generic.hs

```diff
@@ -0,0 +1,43 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Witch.Generic where
+
+import qualified GHC.Generics as Generics
+import qualified Witch.From as From
+
+class GFrom s t where
```

Comment:
This is all implementation details. End users wouldn't need to know or care about `GFrom`.

Reply:
That being said, I should either hang some documentation on this or on the `From s (Generically t)` instance.